scripts/test-functional.sh

```diff
@@ -37,16 +38,20 @@ pushd $ML_CORE_TEST_HARNESS_DIR
 
   ## Start the test harness
   echo "==> Starting Docker compose"
-  docker compose --project-name ttk-func --ansi never --profile all-services --profile ttk-provisioning --profile ttk-tests up -d
+  ## For some reason the env vars are not overriding the .env file found in the ttk repo
+  ## So we're setting it in the command
```

Comment:
Did you try with export?

Reply:
Sure let me try now.

Reply:
![image](https://github.com/mojaloop/account-lookup-service/assets/5932442/91d4776f-f2ec-43d7-ba3e-d60bf214f05c)
Looks good.